enhancements/update/eus-upgrades-mvp.md

```diff
@@ -393,6 +390,31 @@ The idea is to find the best form of an argument why this enhancement should _no
 
 ## Alternatives
 
+### MCO Kubelet Skew Enforcement
+
+Instead of the API Server Operator enforcing kubelet skew compliance through
+the `Upgradeable` flag, the MCO could provide this functionality. Either of
+these two operators are the obvious choice for such a check since they are
+responsible for both halves of the kubelet-API Server interaction. It makes
+more sense for the leading component to implement the check, however, since
+it's the leading edge that's going to violate the skew compliance first. In the
+case of OpenShift, that leading edge is the API Server and it makes more sense
+for it to determine whether a step forward is going to violate the skew policy.
+On top of that, the gating mechanism we have today is the `Upgradeable=False`
+flag, which indicates that a particular operator cannot be upgraded, thereby
+halting the upgrade of the entire cluster. It doesn't make sense for the MCO to
+assert this condition, since an upgrade of the MCO would actually reduce the
+skew. If the MCO were to use this mechanism to enforce the skew, it would be a
+reinterpretation of the function of that flag to instead indicate that the
+entire cluster cannot be upgraded. It's a subtle but important distinction that
```

Comment:
I'm interested in how placing this condition on the kube-apiserver-operator is a better experience for users when compared with the locality benefits to users outlined https://github.com/openshift/enhancements/pull/762/files#r623192675 .